src/containers/MathObjects/components/StatusSymbol/components/ColorPickerPopover.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Popover } from 'antd'
+import ColorPicker from './ColorPicker'
+
+const colors = [
+  '#1abc9c',
+  '#2ecc71',
+  '#3498db',
+  '#9b59b6',
+  '#8e44ad',
+  '#2c3e50',
+  '#f1c40f',
+  '#e67e22',
+  '#e74c3c',
+  '#f39c12'
+]
+
+export default class ColorPickerPopover extends React.Component {
+
+  static propTypes = {
+    visible: PropTypes.bool.isRequired,
+    onPickColor: PropTypes.func.isRequired,
+    onHideColorPicker: PropTypes.func.isRequired,
+    children: PropTypes.element.isRequired
+  }
+
+  swatch = (
```

Comment:
@stardust66 Since you changed the name on Swatch component, could you change this variable name? Maybe this.colorPicker (reflects what it is) or this.popoverContent (reflects its purpose). Up to you.

After that, I'm ready to merge!

Reply:
Oops